torch/csrc/jit/python/init.cpp

```diff
@@ -828,6 +828,10 @@ void initJITBindings(PyObject* module) {
       .def(
           "wait",
           &PythonFutureWrapper::wait,
+          py::call_guard<py::gil_scoped_release>())
+      .def(
+          "_then",
```

Comment:
Why `_then` instead of just `then`? Is this some convention in JIT?

Reply:
No, not sure if the current signature is appropriate, but need to get this in earlier to unblock fixing profiler tests. So mark it `_then` with no API doc for now, and will create a PR on top to expose this API publicly.